src/Services/validators.ts

```diff
@@ -10,6 +12,14 @@ export function stringLengthIsValid(
   } else return true;
 }
 
+export function numberLengthIsValid(
+  number: number,
+  minLength: number,
+  maxLength: number
+): boolean {
+  return number.toString().length < minLength || number.toString().length > maxLength ? false : true 
```

Comment:
You could actually utilise your string length function...

```suggestion
  return stringLengthIsValid(number.toString(), minLength, maxLength) 
```

Reply:
done